app/concepts/reservations/use_cases/pay.rb

```diff
@@ -1,14 +1,14 @@
 module Reservations
   module UseCases
-    class Create
+    class Pay
       attr_reader :repository
 
       def initialize(repository: Reservations::Repository.new)
         @repository = repository
       end
 
-      def call(params:)
-        repository.create(params)
+      def call(id:)
+        repository.update(id, :status => "paid")
```

Comment:
```suggestion
        repository.update(id, status: "paid")
```

Let's use the newer Ruby notation for hashes